store/etcd/rulesets_test.go

```diff
@@ -129,7 +129,7 @@ func TestList(t *testing.T) {
 		createRuleset(t, s, "b", rs)
 		createRuleset(t, s, "a", rs)
 
-		paths := []string{"a/1", "a", "b", "c"}
+		paths := []string{"a", "a/1", "b", "c"}
```

Comment:
This shouldn't be changed lightly, this ensures the ordering is always respected. If we change that there must be a good reason to do so. What is the reasoning behind that change?

Reply:
Does that mean the ordering will be different if we want to display all the versions? or if we use `pathOnly`?

Reply:
The ordering is different when the queries are based on the entries namespace.
Thus, it will be different if we list all versions. But for the `pathOnly` and the default listing (only latest version), the ordering will be the same.